client/my-sites/upgrades/checkout/privacy-protection-dialog.jsx

```diff
@@ -49,7 +41,9 @@ module.exports = React.createClass( {
 					<h1>{ this.translate( 'Why do I need Privacy Protection?' ) }</h1>
 					<p>
 						{ this.translate( 'Domains are required to have publicly accessible contact information.' ) }
-						<span className="line-break">{ this.translate( 'With Privacy Protection, we show our partner\'s contact information instead of your own, helping to:' ) }</span>
+						<span className="line-break">
+							{ this.translate( 'With Privacy Protection, we show our partner\'s contact information instead of your own, helping to:' ) }
```

Comment:
We should probably change this one. It's not `partner` anymore for all the privacy info. Some is our own.